apps/site/assets/ts/stop/components/AccessAmenitiesModal.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Alert, Facility } from "../../__v3api";
+import Alerts from "../../components/Alerts";
+import { hasFacilityAlert } from "../../models/alert";
+import { AmenityModal, AmenityLink } from "./amenities/AmenityCard";
+
+const AccessAmenitiesModal = ({
+  stopName,
+  alerts,
+  facilities,
+  facilityType
+}: {
+  stopName: string;
+  alerts: Alert[];
+  facilities: Facility[];
+  facilityType: "Elevators" | "Escalators";
+}): JSX.Element => {
+  const hasFacilities = facilities ? facilities.length > 0 : false;
+  const linkText =
+    facilityType === "Elevators"
+      ? "Report an elevator issue"
+      : "Report an escalator issue";
+  return (
+    <>
+      {hasFacilities && (
+        <AmenityModal headerText={`${facilityType} at ${stopName}`}>
+          <Alerts alerts={alerts} />
+          <h2>Eleavator Status</h2>
+          <div className="access-amenities-row facilities-list-header">
+            <h3>Elevator</h3>
+            <h3>Status</h3>
```

Comment:
I'm not sure using heading tags in this particular location makes sense given no text following them.. What was the rationale for not using a `<table>` for this?